column/api/objects/__init__.py

```diff
@@ -0,0 +1,3 @@
+class State(object):
+    RUNNING = "RUNNING"
```

Comment:
In the api side currently, it can not get the state of ansible run. It will involve changes of column library to provide the interface to get the  run state. 

i would like to put it into the future task